### PR TITLE
Run xcode 10 migration wizard

### PR DESCRIPTION
**Pull request by edelabar**
Ran the Xcode 10 migration wizard, no code changes.

We use this project as a submodule off a tag, it would be greatly appreciated if you could merge this a create a new tag from it so we can kill the Xcode warnings. Thanks!

### CODE REVIEW

File: AppAuth.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2036,7 +2036,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
```
Comment (diejmon) [inline AppAuth.xcodeproj/project.pbxproj]: What's the reason to bump deployment target?

Comment (edelabar) [reply AppAuth.xcodeproj/project.pbxproj]: No reason, the wizard did it. The Xcode 10 ui only has down to 8.0 in the dropdown but you can type-in whatever version you want. I reverted it to 7.0.

**Comment by WilliamDenniss** [comment]
Can you sign the [CLA](https://openid.net/intellectual-property/contribution-license-agreement/), or if you have already point me to it?

**Comment by edelabar** [comment]
I just e-signed it and emailed it to the address on that page.

**Comment by edelabar** [comment]
http://openid.net/wordpress-content/uploads/2018/09/Eric-DeLaber_CLA-Agreement-Individual_2018-09-18.pdf

**Pull request merged**